### ISSUE TITLE
Arcade 2.6 incompatibility with Python 3.7

### ISSUE THREAD

**Issue by caffeinepills** [pythonarcade/arcade#934]
## Bug Report

Actual behavior: 
```
Traceback (most recent call last):
  File "C:\Users\Admin\Desktop\arcade_test.py", line 1, in <module>
    import arcade
  File "C:\Python\Python37\lib\site-packages\arcade\__init__.py", line 290, in <module>
    from arcade import experimental
  File "C:\Python\Python37\lib\site-packages\arcade\experimental\__init__.py", line 5, in <module>
    from .shadertoy import Shadertoy, ShadertoyBuffer
  File "C:\Python\Python37\lib\site-packages\arcade\experimental\shadertoy.py", line 38
    self._mouse_pos: Tuple[float, float] = 0.0, 0.0
                                              ^
SyntaxError: invalid syntax

```

Currently getting the above error.

### Steps to reproduce/example code: Install latest using pip3 or pip.

Also note I do get the following multiple (6) times during installation: `WARNING: Ignoring invalid distribution -illow (c:\python\python37\lib\site-packages)`. Not sure if that's related.

Versions:
OS: Windows 10
Python: Python 3.7 64 bit
Arcade: 2.6

**Comment by pvcraven** [comment]
This appears to be 3.8+ syntax. We either need to drop 3.7 support and give a proper warning, or test for Python 3.7 compatibility.

**Comment by Cleptomania** [comment]
I can't seem to find anything about this in the 3.8 changelog, can you point out exactly what's wrong there? Does it just need parentheses to declare the tuple?

**Comment by pvcraven** [comment]
I think it is the typing of the self attributes, but I'm not sure.

**Comment by caffeinepills** [comment]
Ah, ok, thought it was compatible as the docs currently say 3.6 or newer: https://api.arcade.academy/en/latest/install/index.html

**Comment by einarf** [comment]
We might need to do some cleanup related to this. I know there are several places with such type annotations. They were added in 3.8 as far as I know.

**Comment by pvcraven** [comment]
Fixed with 2.6.1
